test/functional/rpc_generateblock.py

```diff
@@ -6,26 +6,29 @@
 '''
 
 from test_framework.test_framework import BitcoinTestFramework
+from test_framework.address import ADDRESS_BCRT1_P2WSH_OP_TRUE
 from test_framework.util import (
     assert_equal,
     assert_raises_rpc_error,
 )
+from test_framework.wallet import MiniWallet
 
 
 class GenerateBlockTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
 
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
-
     def run_test(self):
         node = self.nodes[0]
+        miniwallet = MiniWallet(node)
+        address = ADDRESS_BCRT1_P2WSH_OP_TRUE
```

Comment:
nit: I'm not sure if this really matters, but would it be better to use `miniwallet._address`? I know they are hard-coded to the same value rn

@MarcoFalke what's you're opinion on this, would it be beneficial to add a getter to `MiniWallet`?
```python
@property
def address(self):
  return self._address
```

Reply:
I think it is fine to just make `_address` public (remove the underscore)